### DISCUSSION TITLE
Max-Min methods on Deta Base

### DISCUSSION THREAD

**Discussion by afelro97** [deta/cloud-docs#585] [Help]
I have a table where I need to extract the records ordered by a field in descending order to be able to generate a top 10, top 20, top 50. What is the best way to avoid having to download the entire table and manually summarize because I do not see operators in Deta Base SDK ?

**Comment by sudo-subg** [comment]
There are operators in Deta Base. https://docs.deta.sh/docs/base/queries#operators
